autoPyTorch/pipeline/components/setup/network_backbone/utils.py

```diff
@@ -103,13 +103,14 @@ def shake_get_alpha_beta(
         return result if not is_cuda else (result[0].cuda(), result[1].cuda())
 
     # TODO implement other update methods
-    if method != 'even-even':
-        alpha = torch.rand(1)
-    else:
+    if method == 'even-even':
         alpha = torch.FloatTensor([0.5])
-    if method in ['shake-shake', 'shake-drop']:
+    else:
+        alpha = torch.rand(1)
+
+    if method == 'shake-shake':
```

Comment:
https://github.com/automl/Auto-PyTorch/blob/292178197068a59a6a9c2ec869349dfec0ffd96f/autoPyTorch/pipeline/components/setup/network_backbone/ResNetBackbone.py#L347, you should change the following here too for 'shake_drop' and pass the config.

Reply:
oh yeah sorry. Thanks for the catch